releasewarrior/helpers.py

```diff
@@ -8,6 +8,9 @@
 
 logger = logging.getLogger('releasewarrior')
 
+BUG_RE = r'(?<!\[)\b[bB][uU][gG]\s*(\d+)(?!\]\(http)\b'  # Don't match already linked bugs
```

Comment:
tyvm. so looking at your repl test strings, this will work so long as you don't wrap `bug 1234` with square brackets? I wonder if your `\(http)` is doing anything?